reference/configuration/framework.rst

```diff
@@ -1,21 +1,26 @@
 .. index::
     single: Configuration reference; Framework
 
-FrameworkBundle Configuration ("framework")
-===========================================
+FrameworkBundle Configuration
+=============================
```

Comment:
Do you not like the `("framework")` part? I always like it... because what the user *really* thinks is "what things can I put under this `framework` key?)